src/modules/logicModule/LM_PC_RecurringPayments_v1.sol

```diff
@@ -137,6 +138,13 @@ contract LM_PC_RecurringPayments_v1 is
         }
 
         emit EpochLengthSet(newEpochLength);
+
+        // Set the flags for the PaymentOrders (this module uses 2 flags).
+        uint8[] memory flags = new uint8[](2);
+        flags[0] = 1; // start, flag_ID 1
```

Comment:
since you will use these values ("0", "1", "2" etc) in different places with a fixed meaning, it would be a good idea to introduce constants here for better readability and a lower chance of making mistakes. So you'd have a `const FLAG_ORDERID = 0` etc, and you could call this here `__ERC20PaymentClientBase_v1_init([FLAG_START, FLAG_END])`